apps/lfx-one/src/app/modules/dashboards/executive-director/components/event-growth-drawer/event-growth-drawer.component.html

```
@@ -0,0 +1,248 @@
+            <span class="text-2xl font-semibold text-gray-900">${{ data().revenuePerAttendee | number: '1.2-2' }}</span>
+          } @else {
+            <span class="text-2xl font-semibold text-gray-400">—</span>
+          }
+        </div>
+      </lfx-card>
+      <lfx-card styleClass="flex-1">
+        <div class="flex flex-col gap-1">
+          <span class="text-sm text-gray-500">Revenue YoY</span>
+          @if (data().revenueMomChange !== 0) {
+            <span class="text-2xl font-semibold" [class.text-green-600]="data().revenueMomChange > 0" [class.text-red-600]="data().revenueMomChange < 0">
+              {{ data().revenueMomChange > 0 ? '+' : '' }}{{ data().revenueMomChange.toFixed(1) }}%
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)